### ISSUE TITLE
A11y P2: Recent Episodes - ZoomText AppRdr tool only reads out 'audio' and the release date for each item

### ISSUE THREAD

**Issue by LilyL0u** [bbc/simorgh#8355]
When using ZoomText on IE11 (as it should be) and using the AppRdr tool, as the reader reads through the recent episode items it only reads out 'audio' and the release date for each of them.

@jroebu14 also experienced this on his a11y laptop.

**Comment by jroebu14** [comment]
@LilyL0u this one is fixed isn't it??

**Comment by LilyL0u** [comment]
Just retested, this is now fixed
